network-conduit-tls/ChangeLog.md

```diff
@@ -1,3 +1,7 @@
+## 1.3.2
+
+* Drop SSLv3 from supported versions.
```

Comment:
How about clarifying that this simply uses the default upstream value, since this comment may fall out of sync in the future with changes to the `tls` library.

Reply:
Done!